wsu-extended-gravity-forms.php

```diff
@@ -142,5 +143,23 @@ public function remove_export_fields( $form ) {
 
 		return $form;
 	}
+
+	/**
+	 * Gravity Forms uses `chmod` after renaming a temporary file to ensure that
+	 * permissions are set to 0644 by default. When S3 Uploads is enabled, files
+	 * are hosted on AWS S3 and `chmod()` is not a supported stream operation in
+	 * the AWS SDK that S3 Uploads uses. In these cases, we're okay to skip `chmod()`.
+	 *
+	 * @param string $permission
+	 *
+	 * @return bool|string
+	 */
+	public function file_permission( $permission ) {
+		if ( function_exists( 's3_uploads_enabled') && s3_uploads_enabled() ) {
```

Comment:
The spacing error here - `enabled')` is fixed in #17 as part of applying PHPCS :)